eslint.config.mjs

```diff
@@ -0,0 +1,178 @@
+import { fixupPluginRules } from "@eslint/compat";
+import js from "@eslint/js";
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import tsParser from "@typescript-eslint/parser";
+import importt from "eslint-plugin-import";
+import prettier from "eslint-plugin-prettier/recommended";
+import simpleImportSort from "eslint-plugin-simple-import-sort";
+import * as espree from "espree";
+import globals from "globals";
+
+export default [
+  {
+    ignores: [".yarn/", "packages/*/build/", "packages/*/dist/", "packages/*/docs/", "packages/*/examples/"],
+  },
+  js.configs.recommended,
+  prettier,
+  // importt.flatConfigs.recommended,
+  {
+    plugins: {
+      "simple-import-sort": simpleImportSort,
+      import: fixupPluginRules(importt),
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.jasmine,
+        ...globals.node,
+        ...globals.worker,
+      },
+
+      parser: espree,
```

Comment:
As espree is the default parser (https://eslint.org/docs/v8.x/use/configure/parser) I think we can just remove this setting and remove the explicit `espree` + `@types/espree` dependency, right?